decaf377-fmd/src/clue_key.rs

```diff
@@ -73,8 +73,9 @@ impl ExpandedClueKey {
         Ok(())
     }
 
-    /// Create a [`Clue`] intended for the [`DetectionKey`](crate::DetectionKey)
-    /// corresponding to this clue key.
+    /// Create a [`Clue`] deterministically. Intended for the
+    /// [`DetectionKey`](crate::DetectionKey) corresponding to this clue key based
+    /// on a random seed.
```

Comment:
I think this would be clearer phrased as:

> Create a [`Clue`] intended for the [`DetectionKey`] corresponding to this clue key, deterministically, using the provided random seed.